app/_src/guides/meshidentity-guide.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Issuing Identity with MeshIdentity
+---
+
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-system = site.mesh_namespace | default: "kuma-system" %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma-control-plane = kuma | append: "-control-plane" %}
+
+The [MeshIdentity](TODO) allows you to issue identity for selected data planes. This approach is [Spiffe](https://spiffe.io/docs/latest/spiffe-about/overview/) compliant and can
+be used with Spire. In this guide we will take a look on how to issue identities using bundled provider.
+
+## Prerequisites
+
+Before you begin, make sure you have the following tools installed:
+
+* [Helm](https://helm.sh/) – used to install and manage Kubernetes applications
+* [minikube](https://minikube.sigs.k8s.io/docs/) – used to run a local Kubernetes cluster for testing
+
+### Start a Kubernetes cluster
+
+Start a local Kubernetes cluster using minikube. The `-p` flag creates a new profile named `mesh-zone`:
+
+```bash
+minikube start -p mesh-zone
+```
+
+{% tip %}
+If you already have a running Kubernetes cluster, either locally or in the cloud (for example, EKS, GKE, or AKS), you can skip this step.
+{% endtip %}
+
+### Install {{ Kuma }}
+
+Install {{site.mesh_product_name}} control plane with Helm by executing:
+
+```sh
+helm repo add {{site.mesh_helm_repo_name}} {{site.mesh_helm_repo_url}}
+helm repo update
+helm install --create-namespace --namespace {{ kuma-system }} {{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+### Deploy demo application
+
+1.  Deploy the application
+    ```sh
+    kubectl apply -f kuma-demo://k8s/000-with-kuma.yaml
```

Comment:
I'm not sure if this example has `meshServices.mode: Exclusive` but I think you should explicitly mention in the guide that MeshServices are required.